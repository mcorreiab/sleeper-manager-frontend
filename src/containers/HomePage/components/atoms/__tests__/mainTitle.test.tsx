import renderer from 'react-test-renderer';
import MainTitle from "../mainTitle";

it("should render the styled main title", () => {
  const tree = renderer.create(<MainTitle />).toJSON();
  expect(tree).toMatchSnapshot();
});
