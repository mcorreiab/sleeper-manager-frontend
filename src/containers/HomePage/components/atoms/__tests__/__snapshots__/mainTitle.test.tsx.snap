// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the styled main title 1`] = `
<h2
  className="font-normal text-[2.4375rem] md:text-[2.85rem]"
>
  Tool to easy your life at 
  <span
    className="text-[#f26506]"
  >
    sleeper
  </span>
   
  fantasy platform
</h2>
`;
